components/admin_console/admin_definition.jsx

```diff
@@ -2368,6 +2368,17 @@ const AdminDefinition = {
                         help_text_default: 'Display a preview of website content, image links and YouTube links below the message when available. The server must be connected to the internet and have access through the firewall (if applicable) to the websites from which previews are expected. Users can disable these previews from Account Settings > Display > Website Link Previews.',
                         isDisabled: it.not(it.userHasWritePermissionOnResource('site')),
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_TEXT,
+                        key: 'ServiceSettings.RestrictLinkPreviews',
+                        label: t('admin.customization.restrictLinkPreviewsTitle'),
+                        label_default: 'Disable link previews from these domains:',
+                        help_text: t('admin.customization.restrictLinkPreviewsDesc'),
+                        help_text_default: 'Link previews and image link previews will not be shown for the above list of comma-separated domains.',
+                        placeholder: t('admin.customization.restrictLinkPreviewsExample'),
+                        placeholder_default: 'E.g.: "corp.mattermost.com, mattermost.org"',
+                        isDisabled: it.not(it.userHasWritePermissionOnResource('site')),
```

Comment:
@esethna  should this also be disabled if `Enable Link Previews` is disabled? I've looked into the ticket but it is not specified there, but it might make some sense to have that dependency

Reply:
@Willyfrog @berkeka ya good point. It should be in a disabled state (greyed out), if link previews are disabled

Reply:
@Willyfrog @esethna Implemented it with this 09e91e3 commit.